services/121-service/src/cronjob/cronjob.service.ts

```diff
@@ -25,7 +38,16 @@ export class CronjobService {
     const accessToken = await this.axiosCallsService.getAccessToken();
     const url = `${this.axiosCallsService.getBaseUrl()}/financial-service-providers/intersolve-voucher/cancel`;
     const headers = this.axiosCallsService.accesTokenToHeaders(accessToken);
-    await this.httpService.post(url, {}, headers);
+    const response: AxiosResponse = await this.httpService.post(
+      url,
+      {},
+      headers,
+    );
+    if (response.status < 200 || response.status >= 300) {
+      throw new Error(
+        `Failed to run Cron Job cronCancelByRefposIntersolve. Status code: ${response.status}`,
```

Comment:
You could consider making a method that does this. Something like:

`throwIfResonceCodeNotSucces(responseCode, cronjobName: string)`

Reply:
Try to filter out the private method, or put it outside of the class.

Reply:
Done.